verdande-core/src/main/scala/org/verdande/core/Histogram.scala

```diff
@@ -102,15 +95,16 @@ object Histogram {
     if (buckets.last != Double.PositiveInfinity) {
       throw new IllegalArgumentException("buckets doesn't end with Positive Infinity")
     }
-    buckets.zip(buckets.tail).foreach { case (first, second) =>
-      if (first >= second) {
-        throw new IllegalArgumentException("bucket is not monotonically increasing")
-      }
+    buckets.zip(buckets.tail).foreach {
+      case (first, second) =>
```

Comment:
I don't think here have another now line have large benefits - here type is know pattern matching is use more about destructering rather then clasical case matching.